src/components/form/multichoice/MultiChoice.tsx

```
@@ -9,6 +9,8 @@ export interface MultiChoiceInputProps {
   entry?: StringEntry;
   onChange?: (values: string[]) => void;
   helpText?: JSX.Element;
+  maxItems?: Number;
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)